.ci/setup_kong.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+# download Kong deb
+
+sudo apt-get update
+sudo apt-get install openssl libpcre3 procps perl wget zlibc
+
```

Comment:
Can we switch to Docker instead?
That would help with keeping CI and localhost using the same setup.

Reply:
Sure, makes sense